### PR TITLE
formula: advance 22.2 to 22.2.19

### PR DESCRIPTION
**Pull request by abarganier**

### CODE REVIEW

File: Formula/cockroach@22.2.rb [hunk 1]
```diff
@@ -4,15 +4,15 @@
 class CockroachAT222 < Formula
   desc "Distributed SQL database"
   homepage "https://www.cockroachlabs.com"
-  version "22.2.17"
+  version "22.2.19"
```
Comment (celiala) [inline Formula/cockroach@22.2.rb:7]: confirming that this is what we want!
(i forgot to brew update for 22.2.18 - thanks for bumping to 22.2.19!)

File: Formula/cockroach@22.2.rb [hunk 2]
```diff
@@ -31,9 +31,15 @@ def install
         "#{lib}/cockroach/libgeos.dylib", "#{lib}/cockroach/libgeos.dylib"
       system "install_name_tool", "-id",
         "#{lib}/cockroach/libgeos_c.1.dylib", "#{lib}/cockroach/libgeos_c.dylib"
-      system "install_name_tool", "-change",
-        "@rpath/libgeos.3.8.1.dylib", "#{lib}/cockroach/libgeos.dylib",
-        "#{lib}/cockroach/libgeos_c.dylib"
+      if version < Version::new("23.2.0")
```
Comment (celiala) [inline Formula/cockroach@22.2.rb:34]: confirming that this is what we want as well!

**Review by celiala — APPROVED** [review]
LGTM! thank you!

**Comment by abarganier** [comment]
TFTR!

**Pull request merged**